src/util/Constants.js

```diff
@@ -119,10 +131,23 @@ function makeImageUrl(root, { format = 'webp', size } = {}) {
  * @property {number} [size] One of `16`, `32`, `64`, `128`, `256`, `512`, `1024`, `2048`, `4096`
  */
 
+/**
+ * Options for Sticker Image URL.
+ * @typedef {Object} StickerURLOptions
+ * @property {number} [size] One of `16`, `32`, `64`, `128`, `256`, `512`, `1024`, `2048`, `4096`
+ * @property {boolean} [passthrough=true] If false, the image will make sticker static.
```

Comment:
> the image will make sticker static

this doesn't sound right, perhaps 'the image will be static'?

Reply:
maybe, to be more specific:
```suggestion
 * @property {boolean} [passthrough=true] Whether the image will be animated if it has an animated version.
```
and maybe rename the property to `animated` for clarity?